lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -26,8 +26,7 @@ updateScriptTags = (activeDocument, newScripts, callback) ->
   )
 
 extractTrackedAssets = (doc) ->
-  for node in doc.head.children when node.dataset.turbolinksTrack?
-    node
+  [].slice.call(doc.querySelectorAll('[data-turbolinks-track]'))
```

Comment:
Does this do anything that [the original turbolinks fix](https://github.com/turbolinks/turbolinks-classic/blob/master/lib/assets/javascripts/turbolinks.coffee#L352) doesn't do?